### DISCUSSION TITLE
Error in /lang

### DISCUSSION THREAD

**Discussion by thiagocatini** [arnoudkooi/SN-Utils#307] [General]
Hi,
You asked me in FireFox addons to report the /lang problem here, follow the print.

Remembering that I'm inside a Catalog Item and my language is pt-br or /pb and when I put it to change the ENTER it doesn't work inside.

![image](https://user-images.githubusercontent.com/104521258/192796879-909ada48-f571-472e-95e4-8d70409d187c.png)

![image](https://user-images.githubusercontent.com/104521258/192797891-bc49819b-aca0-48df-a789-0784c5c6378a.png)

Example: Where is NAME should appear NOME ( in pt-bt ) after giving /lang pb
Catalogs = Catálogos
Category = Categoria

Resume: When I press enter it doesn't work inside a catalog. Everything else works x.x

**Comment by arnoudkooi** [comment]
Do you see a console error? Maybe easiest to setup a quick call to investigate

**Comment by thiagocatini** [comment]
Hi,
The console error:
Before Enter:
![image](https://user-images.githubusercontent.com/104521258/192831795-97944757-11c7-4646-9046-d5a885d74207.png)

After Enter:
![image](https://user-images.githubusercontent.com/104521258/192831994-6fbb21d7-d526-4e14-a4ca-d01bf0e4ed3f.png)

ps: Remembering that my language is pt-br "/lang pb" and sometimes I need to change between /lang en for /lang pb and in the catalogs the page does not update with the language. The Utils box is up there standing

**Comment by arnoudkooi** [comment]
Thanks, I just published an update. Could you try and update to version 6.1.5.1 The FireFox version should be available now, or in a minute. Be sure to refresh the ServiceNow page as well after updating

**Comment by thiagocatini** [comment]
![image](https://user-images.githubusercontent.com/104521258/192845224-1942b4f3-1069-4284-8300-3acc60908aca.png)

Okay, I did everything you asked. The error screen appears to be smaller now, but it still doesn't refresh the page for the selected language.

And this erros is just in the FireFox version x.x

**Comment by arnoudkooi** [comment]
Im sorry, Im not able to test myself, so I hope the newest fix will work. Version 6.1.5.2 should now be available.

**Reply by thiagocatini** [reply]
> Im sorry, Im not able to test myself, so I hope the newest fix will work. Version 6.1.5.2 should now be available.
Hi, can you reverse to 6.1.5.1? Just for more one test
![image](https://user-images.githubusercontent.com/104521258/192862247-08b08b9a-88ab-4158-b606-37c81731b94e.png)

Now, this error appears:
![image](https://user-images.githubusercontent.com/104521258/192862276-2545daf0-11de-4e25-a265-220ef656cea1.png)

**Comment by arnoudkooi** [comment]
The 1th screenshot is not related to SN Utils.
Do other functions work as expected, like /sa /spw and so on?

**Reply by thiagocatini** [reply]
> The 1th screenshot is not related to SN Utils. Do other functions work as expected, like /sa /spw and so on?

Yep. its all Ok with other functions

**Comment by arnoudkooi** [comment]
Can you check the g_ck in value in console like below?

```
g_ck // 'null' check if it is a string if so, set it to null
g_ck = null

```
after that, try again..

BTW:
I suspect your system property `glide.security.use_csrf_token` is set to `false`, probably best would be to set it to `true`

**Reply by thiagocatini** [reply]
Hi,
As I understand it, it is translating, but now it is duplicating the catalog within itself
![image](https://user-images.githubusercontent.com/104521258/192872424-8051197d-3c22-46be-9d04-980720a5e76b.png)

![image](https://user-images.githubusercontent.com/104521258/192872617-6933fe3c-fddf-418d-98c0-cb2e3149396c.png)

Maybe u can reverse all versions. We try xD... Thanks so much

**Comment by arnoudkooi** [comment]
I did one last update for today.
Lets try to connect tomorrow and setup a quick call

**Comment by arnoudkooi** [comment]
Is this now fixed?

**Reply by thiagocatini** [reply]
Yes. I'm very grateful!

Success, att.

**Reply by arnoudkooi** [reply]
Good! Have you revisited your FF review?

**Reply by thiagocatini** [reply]
Yes, I edited.
